docs/rewards/reward-boosts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: reward-boosts
+title: Rewards Boosts
+sidebar_label: Rewards Boosts
+slug: /project/rewards-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+## Overview
+In addition to the **Base Rewards**, a device already gets for contributing to weather data during the regular daily distribution. Learn more about the daily rewards [Reward Mechanism (RM) ](/reward-mechanism).
```

Comment:
I think a sentence is missing here before "In addition" e.g. "Reward boosts are rewards......"